src/logic/operate.js

```diff
@@ -0,0 +1,28 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+
+  if (operation === '÷') {
+    return one.div(two).toString();
+  }
```

Comment:
- [ ] When I divide a `number` with `zero` let us say `5/0` normally you have to get an `error` in the screen but currently am getting this
![e](https://user-images.githubusercontent.com/44745855/130265733-23b540ea-69f9-40c8-9aaa-c6f03bdf8668.png)
you can check in your calculator what am talking about..please fix that